code/modules/uplink/uplink_items.dm

```diff
@@ -2242,14 +2242,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "A deadly energy saw. Comes in a slick black finish."
 	cost = 5
 	item = /obj/item/melee/transforming/energy/sword/esaw
-	restricted_roles = list("Medical Doctor", "Chief Medical Officer", "Paramedic", "Brig Physician")
+	restricted_roles = list("Medical Doctor", "Chief Medical Officer", "Brig Physician")
```

Comment:
Convert this to defines while you're here

Reply:
This would involve converting the entirity of the uplink restricted roles into defines in a PR about removing paramedic. It would be too out of scope to do in this PR, especially considering the potentially contraversial nature of it.